components/Points.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+class Points extends React.Component {
+  constructor (props, context) {
+    super(props, context)
+    this.state = {count: this.props.count}
+  }
+  render () {
+    return (
+      <div
+        onClick={this.props.onClick}
+      >{`${this.props.count}`}</div>
```

Comment:
You actually don't need the extra interpolation here. `>{this.props.count}<` would be enough.